epoch_failover_test.go

```diff
@@ -387,17 +367,18 @@ func createEmptyVote(md ProtocolMetadata, signer NodeID) *EmptyVote {
 	return emptyVoteFrom2
 }
 
-func waitForEvent(t *testing.T, start time.Time, e *Epoch, events chan struct{}) {
-	now := start
+func waitForBlockProposerTimeout(t *testing.T, e *Epoch, startTime time.Time) {
```

Comment:
I think it's better to use the approach from `assertNotarization` instead of this one.